src/components/editor/metaform-editor-right-drawer.tsx

```diff
@@ -73,11 +85,265 @@ const MetaformEditorRightDrawer: React.FC<Props> = ({
+      return (
+        <Typography>{ strings.draftEditorScreen.editor.features.selectComponent }</Typography>
+      );
+    }
+    if (sectionIndex !== undefined && fieldIndex === undefined) {
+      const section = pendingForm.sections![sectionIndex];
+      return (
+        <TextField
+          value={ section.title }
+          label={ strings.draftEditorScreen.editor.features.sectionTitle }
+          onChange={ event => updateFormSection({ ...section, title: event.target.value }) }
+        />
+      );
+    }
+
+    if (fieldIndex !== undefined && sectionIndex !== undefined) {
+      const field = pendingForm.sections![sectionIndex].fields![fieldIndex];
+      if (field === undefined) {
+        return null;
+      }
+      if (field !== undefined) {
+        const disableForm = field.type === "select" || field.type === "date-time" || field.type === "radio" || field.type === "checklist" || field.type === "date";
+
+        return (
+          <>
+            { renderFieldsCategorytitle(strings.draftEditorScreen.editor.features.fieldDatas) }
+            <TextField
+              label={ strings.draftEditorScreen.editor.features.fieldTitle }
+              value={ field.title ? field.title : " " }
+              onChange={ event => updateFormField({
+                ...field, title: event.target.value, name: `${sectionIndex}.${fieldIndex}`
+              }) }
+            />
+        
+            <Divider/>
+            { renderFieldsCategorytitle(strings.draftEditorScreen.editor.features.required) }
+            <FormControlLabel
+              label={ strings.generic.yes }
+              control={
+                <Checkbox
+                  value={ field.required || false }
+                  onChange={ event => updateFormField({ ...field, required: event.target.checked }) }
+                />
+              }
+            />
+            <Divider/>
+            <Typography variant="subtitle1" style={{ width: "100%" }}>
+              { strings.draftEditorScreen.editor.features.defineUserGroup }
+            </Typography>
+
+            <FormControl disabled={ !disableForm }>
+              <FormControlLabel
+                label={ strings.generic.yes }
+                control={
+                  <Checkbox
+                    value={ field.required || false }
+                    onChange={ event => updateFormField({ ...field, required: event.target.checked }) }
+                  />
+                }
+              />
+              <Typography fontSize="12px">
+                { strings.draftEditorScreen.editor.features.selectableOnlyOn }
+              </Typography>
+            </FormControl>
+          </>
+        );
+      }
+    }
+  };
+
+  /**
+   * Render visiblity condition selection menu
+   *
+   */
+  const fieldConditionComponent = () => {
+    if (sectionIndex === undefined || fieldIndex === undefined) {
+      return;
+    }
+    if (fieldIndex !== undefined && sectionIndex !== undefined) {
```

Comment:
```suggestion
    if (fieldIndex && sectionIndex) {
```

Reply:
if (fieldIndex && sectionIndex) 

doesnt work for some reason.